config/templates/stack-template.json

```diff
@@ -1037,10 +1027,10 @@
     }
     {{if $.ElasticFileSystemID}}
     ,
-    "{{$subnetLogicalName}}MountTarget": {
+    "{{$subnet.SubnetLogicalName}}MountTarget": {
```

Comment:
this makes a confusing resource name if we use existing subnets. Not sure it is worth fixing though :)